### PR TITLE
techdebt: [CDS-84682]: Remove CDS_HELM_MULTIPLE_MANIFEST_SUPPORT_NG ff

### PR DESCRIPTION
**Pull request by sribalij**
## Describe your changes
Remove CDS_HELM_MULTIPLE_MANIFEST_SUPPORT_NG from code
## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>

- Feature build: `trigger feature-build`
- 
  Specific builds
  - `trigger manager`
  - `trigger dms`
  - `trigger ng_manager`
  - `trigger cvng `
  - `trigger cmdlib`
  - `trigger template_svc`
  - `trigger events_fmwrk_monitor`
  - `trigger event_server`
  - `trigger change_data_capture`
  -  Trigger multiple builds together: For eg: `trigger dms, manager`
- Immutable delegate `trigger publish-delegate`
</details>

## [Latest PR Check Triggers](https://github.com/harness/harness-core/blob/develop/.github/pull_request_template.md)

<details>
  <summary>PR Check triggers</summary>
You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ut0, ut1`

- Compile: `trigger compile`
- CodeformatCheckstyle: `trigger checkstylecodeformat`
  - CodeFormat: `trigger codeformat`
  - Checkstyle: `trigger checkstyle`
- MessageMetadata: `trigger messagecheck`
- File-Permission-Check: `trigger checkpermission`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- Trigger CommonChecks: `trigger commonchecks`
- PMD: `trigger pmd`
- Copyright Check: `trigger copyrightcheck`
- Feature Name Check: `trigger featurenamecheck`
- UnitTests-ALL: `trigger uts`
- UnitTests-0: `trigger ut0`
- UnitTests-1: `trigger ut1`
- UnitTests-2: `trigger ut2`
- UnitTests-3: `trigger ut3`
- UnitTests-4: `trigger ut4`
- UnitTests-5: `trigger ut5`
- UnitTests-6: `trigger ut6`
- UnitTests-7: `trigger ut7`
- UnitTests-8: `trigger ut8`
- UnitTests-9: `trigger ut9`
- CodeBaseHash: `trigger codebasehash`
- CodeFormatCheckstyle: `trigger checkstylecodeformat`
- SonarScan: `trigger ss`
- GitLeaks: `trigger gitleaks`
- Trigger all Checks: `trigger smartchecks`
- Go Build: `trigger gobuild`
- Validate_Reviews: `trigger review`
- Module Dependency Check: `trigger mdc`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### Rebase Instructions:
To rebase your branch onto the latest changes in the target branch, simply use the following command in a comment: `/rebase`

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/56305)

### CODE REVIEW

**Comment by sribalij** [comment]
trigger uts

**Comment by sribalij** [comment]
trigger ut4

**Comment by sribalij** [comment]
trigger uts

**Review by GoelPratyush99 — COMMENTED** [review]
The Feature flag is being used in the UI side as well, If the feature is GAed then only lets remove this FF

**Review by GoelPratyush99 — APPROVED** [review]

File: 956-feature-flag-beans/src/main/java/io/harness/beans/FeatureName.java [hunk 1]
```diff
@@ -114,9 +114,6 @@ public enum FeatureName {
   CDS_HELM_FETCH_CHART_METADATA_NG(
       "Enables option to fetch helm chart details from the Chart.yaml and expose this as expressions. Epic: https://harness.atlassian.net/browse/CDS-58036",
       HarnessTeam.CDP),
-  CDS_HELM_MULTIPLE_MANIFEST_SUPPORT_NG(
```
Comment (abhinavsingh1196) [inline 956-feature-flag-beans/src/main/java/io/harness/beans/FeatureName.java:117]: we remove the enum value only once pipeline service is released to prod, after removing all code usages of the FF. so we should do this enum removal later?

**Review by abhinavsingh1196 — CHANGES_REQUESTED** [review]

File: 956-feature-flag-beans/src/main/java/io/harness/beans/FeatureName.java (hunk 1 above)
Comment (abhinavsingh1196) [reply 956-feature-flag-beans/src/main/java/io/harness/beans/FeatureName.java:117]: cc: @abosii-harness

Comment (sribalij) [reply 956-feature-flag-beans/src/main/java/io/harness/beans/FeatureName.java:117]: @abhinavsingh1196 ,
Can you please help me understand why that would be the case? This is in manager side of code and not pipeline service right?

Comment (abhinavsingh1196) [reply 956-feature-flag-beans/src/main/java/io/harness/beans/FeatureName.java:117]: Looks like this FF should be good, but some FFs which are shared across services have been known to cause problems.
a relevant thread: https://harness.slack.com/archives/C04B88N1HQD/p1687839229511729

Comment (sribalij) [reply 956-feature-flag-beans/src/main/java/io/harness/beans/FeatureName.java:117]: Yes. Thanks @abhinavsingh1196 for bringing this up

**Review by acasian-harness — DISMISSED** [review]

**Review by abosii-harness — DISMISSED** [review]

File: 956-feature-flag-beans/src/main/java/io/harness/beans/FeatureName.java (hunk 1 above)
Comment (abhinavsingh1196) [reply 956-feature-flag-beans/src/main/java/io/harness/beans/FeatureName.java:117]: Sure, approving this PR.

**Review by abhinavsingh1196 — DISMISSED** [review]

**Comment by sribalij** [comment]
trigger uts

**Comment by sribalij** [comment]
trigger uts

**Review by GoelPratyush99 — APPROVED** [review]

**Comment by sribalij** [comment]
trigger messagecheck

**Review by abhinavsingh1196 — APPROVED** [review]

**Comment by sribalij** [comment]
trigger review

**Review by abosii-harness — APPROVED** [review]

**Comment by acasian-harness** [comment]
@sribalij this has a label `do-not-merge` and i see it is merged, can you confirm?

**Pull request merged**